srcpkgs/tofi/template

```diff
@@ -12,7 +12,12 @@ license="MIT"
 homepage="https://github.com/philj56/tofi"
 changelog="https://raw.githubusercontent.com/philj56/tofi/master/CHANGELOG.md"
 distfiles="https://github.com/philj56/tofi/archive/refs/tags/v${version}.tar.gz"
-checksum=0a700f8b0291ef08b3544cd589157393101d52bbd1e7e6aeee8484f9d520a338
+checksum=dd55347b509af786f133def64f3a86f1db258b5e196de80e53d62827bdcc37bc
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	# some tests fail on i686.
```

Comment:
can you be more specific?

Reply:
the only pattern after checking logs and looking at the tests, is that it fails checking options should be invalid

Reply:
not sure if that's correct info or even useful